pkg/tagger/collectors/kubelet_main.go

```diff
@@ -145,9 +145,14 @@ func (c *KubeletCollector) Fetch(entity string) ([]string, []string, []string, e
 func (c *KubeletCollector) parseExpires(idList []string) ([]*TagInfo, error) {
 	var output []*TagInfo
 	for _, id := range idList {
+		entityID, err := kubelet.KubeIDToTaggerEntityID(id)
+		if err != nil {
+			return nil, err
```

Comment:
wondering if we should log the error and continue 🤔 we don't want to ignore all the ids in the list if just one of them causes a parsing error. wdyt?

Reply:
Makes sense. I'll apply the change 👍